src/main/java/unicash/ui/TransactionCard.java

```diff
@@ -159,8 +179,45 @@ private void categoriesStyleFormatter() {
                 .substring(1, categoriesToStringLength - 1);
 
         String categoriesToStringWithHashTag = "#" + trimmedCategoriesToString;
-        categories.setText(categoriesToStringWithHashTag);
-        categories.setStyle(FONT_STYLE_BOLD);
+        firstCategory.setText(categoriesToStringWithHashTag);
+        firstCategory.setStyle(FONT_STYLE_BOLD);
+        firstCategory.setText("testing");
+        firstCategory.setStyle("-fx-background-color: #FFD43E");
```

Comment:
Is this a constant in `Stylesheet`? If it's not yet, this makes aa great candidate for a constant so there isn't dangling changes

Reply:
Actually this part was a adhoc UI test that I missed out, I've removed those lines, thanks for noticing!